docs/USAGE-GUIDE.md

```diff
@@ -115,36 +137,7 @@ setarch i386 ./config -fPIC no-shared     \
         --prefix=`pwd`/../../libcrypto-root/
 ```
 
-## Building s2n-tls with OpenSSL-1.0.2
-
-To build s2n-tls with OpenSSL-1.0.2, do the following:
-
-```shell
-# We keep the build artifacts in the -build directory
-cd libcrypto-build
-
-# Download the latest version of OpenSSL
-curl -LO https://www.openssl.org/source/openssl-1.0.2-latest.tar.gz
-tar -xzvf openssl-1.0.2-latest.tar.gz
-
-# Build openssl libcrypto
-cd `tar ztf openssl-1.0.2-latest.tar.gz | head -n1 | cut -f1 -d/`
-./config -fPIC no-shared no-libunbound no-gmp no-jpake no-krb5              \
-         no-md2 no-rc5 no-rfc3779 no-sctp no-ssl-trace no-store no-zlib     \
-         no-hw no-mdc2 no-seed no-idea enable-ec-nistp_64_gcc_128 no-camellia\
```

Comment:
Note: `enable-ec-nistp_64_gcc_128` was a typo and should be `enable-ec_nistp_64_gcc_128`